.github/workflows/testing.yml

```diff
@@ -0,0 +1,23 @@
+name: Testing
+
+on:
+  push:
+    branches:
+    - master
+  pull_request:
+
+jobs:
+  docker:
+    name: Docker build
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+    - name: Checkout submodules
+      shell: bash
+      run: |
+        AUTH_HEADER="$(git config --local --get http.https://github.com/.extraheader)"
```

Comment:
is the trailing `/` in the config setting necessary? looks ugly ;)

Reply:
https://github.com/actions/checkout#Checkout-submodules

Bad LordAro for judging a book by its cover :P